js/cryptocom.js

```diff
@@ -0,0 +1,1082 @@
+        }
+        let symbol = undefined;
+        if (market !== undefined) {
+            symbol = market['symbol'];
+        }
+        let open = undefined;
+        let change = undefined;
+        let average = undefined;
+        const last = this.safeNumber (ticker, 'a');
+        const relativeChange = this.safeNumber (ticker, 'c');
+        if (relativeChange !== -1) {
+            open = last / this.sum (1, relativeChange);
+            change = last - open;
+            average = this.sum (last, open) / 2;
+        }
+        return {
+            'symbol': symbol,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeNumber (ticker, 'h'),
+            'low': this.safeNumber (ticker, 'l'),
+            'bid': this.safeNumber (ticker, 'b'),
+            'bidVolume': undefined,
+            'ask': this.safeNumber (ticker, 'k'),
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': open,
+            'close': last,
+            'last': last,
+            'previousClose': undefined,
+            'change': change,
+            'percentage': relativeChange * 100,
+            'average': average,
+            'baseVolume': this.safeNumber (ticker, 'v'),
+            'quoteVolume': undefined,
+            'info': ticker,
+        };
+    }
+
+    parseTrade (trade, market = undefined) {
+        //
+        // public/get-trades
+        //
+        // {"dataTime":1591710781947,"d":465533583799589409,"s":"BUY","p":2.96,"q":16.0,"t":1591710781946,"i":"ICX_CRO"},
+        //
+        // private/get-trades
+        //
+        // {
+        //     "side": "SELL",
+        //     "instrument_name": "ETH_CRO",
+        //     "fee": 0.014,
+        //     "trade_id": "367107655537806900",
+        //     "create_time": 1588777459755,
+        //     "traded_price": 7,
+        //     "traded_quantity": 1,
+        //     "fee_currency": "CRO",
+        //     "order_id": "367107623521528450"
+        // }
+        let timestamp = this.safeInteger (trade, 't');
+        if (timestamp === undefined) {
+            timestamp = this.safeInteger (trade, 'create_time');
+        }
+        let symbol = undefined;
+        if (market !== undefined) {
+            symbol = market['symbol'];
+        }
+        let price = this.safeNumber (trade, 'p');
+        if (price === undefined) {
+            price = this.safeNumber (trade, 'traded_price');
+        }
+        let amount = this.safeNumber (trade, 'q');
+        if (amount === undefined) {
+            amount = this.safeNumber (trade, 'traded_quantity');
+        }
+        const cost = price * amount;
+        const side = this.safeString2 (trade, 's', 'side').toLowerCase ();
+        const id = this.safeString2 (trade, 'd', 'trade_id');
+        const type = 'limit'; // all trades are still limit trades
+        const takerOrMaker = undefined; // can't distinguish my deal from someone else's
```

Comment:
in the private there actualy is a `liquidity_indicator`

 `const takerOrMaker = this.safeString(trade, 'liquidity_indicator');`

Reply:
added! Thanks for letting me know about that 👌